Smart4SURE/Resources/HTML/Base.lproj/About.html

```diff
@@ -17,7 +17,9 @@
     <div class="content">
 
         <p>
-    Smart4SURE is a smartphone app and sub-study of the SURE-PD3 trial. It is designed to gather information that will help researchers improve long-term tracking of Parkinson’s disease changes.  The sub-study will compare smartphone measurements (e.g., of movement, voice, memory, etc.) to those made by study doctors and coordinators, and may help determine whether future Parkinson trials could rely in part on smartphone measurements. Enrolling in Smart4SURE is voluntary and if you enroll in this sub-study you may withdraw from it at any time without jeopardizing participation in the SURE-PD3 trial or its prospects for success. Similarly, those who enroll in Smart4SURE can miss scheduled smartphone activities and continue in the sub-study.
+    Smart<i>4</i>SURE is a smartphone app and sub-study of the SURE-PD3 trial. It is designed to gather information that will help researchers improve long-term tracking of Parkinson’s disease changes.  The sub-study will compare smartphone measurements (e.g., of movement, voice, memory, etc.) to those made by study doctors and coordinators, and may help determine whether future Parkinson trials could rely in part on smartphone measurements.</p>
```

Comment:
Parkinson's in one place, Parkinson in another--assuming this is as you got it from them, so just pointing it out, not saying it's wrong...